src/event_creation.py

```diff
@@ -82,9 +83,42 @@ def check(m):
                 [ctx.guild.id, title, link, description, deadline]
             )
 
-            # TODO add assignment to events list
             await ctx.send('Assignment successfully created!')
             await cal.display_events(None)
+        elif button_clicked == 'project':
+            await ctx.send('What is the title of this project?')
+            msg = await BOT.wait_for('message', timeout=60.0, check=check)
+            project_title = msg.content.strip()
+
+            await ctx.send(
+                'Please enter the due date of this project along with the time\n'
+                'Enter in 24-hour format for e.g. an project due at 11:59pm can be inputted '
+                'as 23:59\nEnter in format `MM-DD-YYYY %H:%M`')
+            msg = await BOT.wait_for('message', timeout=60.0, check=check)
```

Comment:
Should we not accept the start date?

Reply:
Does not actually add any importance to the event in the case of the project. Therefore, accepting only due date.